openfe_gromacs/protocols/gromacs_md/md_methods.py

```diff
@@ -556,7 +558,16 @@ def _create(
             stateA
         )
 
-        system_name = "Solvent MD" if solvent_comp is not None else "Vacuum MD"
+        # Raise an error when no SolventComponent is provided as this Protocol
+        # currently does not support vacuum simulations
+        if not solvent_comp:
```

Comment:
```suggestion
        if solvent_comp is None:
```
This is a minor thing but can improve readability / understanding of what the type is.